site/README-JEKYLL.md

```diff
@@ -15,9 +15,8 @@ If you are running a build on Ubuntu you will need the following packages:
 * zlib1g-dev
 * nginx (or apache2)
 
-
 # Local Development
-1. Install Jekyll and plug-ins in one fell swoop. `gem install github-pages`
+1. Install [Jekyll](https://jekyllrb.com) and plug-ins in one fell swoop. `gem install github-pages`
```

Comment:
Feels like with this PR we should just change this to say 'use make site-devel, it will handle the magic for you'.